__device-tests__/helpers/utils.js

```diff
@@ -92,9 +92,11 @@ const setupDriver = async () => {
 			desiredCaps.app = `sauce-storage:Gutenberg-${ safeBranchName }.apk`; // App should be preloaded to sauce storage, this can also be a URL
 		}
 	} else {
-		desiredCaps = _.clone( ios12 );
+		desiredCaps = _.clone( ios );
 		if ( isLocalEnvironment() ) {
 			desiredCaps.app = path.resolve( localIOSAppPath );
+			delete desiredCaps.platformVersion;
+			desiredCaps.deviceName = 'iPhone 11';
 		} else {
 			desiredCaps.app = `sauce-storage:Gutenberg-${ safeBranchName }.app.zip`; // App should be preloaded to sauce storage, this can also be a URL
 		}
```

Comment:
**Not suggesting we need to make this change in this PR**, but as a future improvement, what would you think about moving as much of this "logic" as we can into separate objects in caps (maybe `iosCi` and `iosLocal`), and just having the logic here select the appropriate caps file (and possibly resolving the app path for local runs)? Seems like it would be nice to keep as much of this "logic" in one place as possible. I'm imagining something like:

```
ios = {
	browserName: '',
	platformName: 'iOS',
	os: 'iOS',
	deviceOrientation: 'portrait',
	automationName: 'XCUITest',
	appiumVersion: '1.15.0', // SauceLabs requires appiumVersion to be specified.
	app: undefined, // will be set later, locally this is relative to root of project
};

exports.iosCi = {
	...ios,
	deviceName: 'iPhone 11 Simulator',
	platformVersion: '13.0',
};

exports.iosLocal = {
	...ios,
	deviceName: 'iPhone 11',
};
```

Reply:
I like that suggestion, adding perhaps some comment about why we don't set the `platformVersion` for the local run. That said, and like Sérgio commented over Slack, this can probably be done in a separate PR so to unblock this one. @SergioEstevao , can you open a relevant ticket? Thanks!